code/core/api/core.api

```diff
@@ -138,10 +155,11 @@ public abstract class com/adobe/marketing/mobile/ExtensionApi {
 	public abstract fun createSharedState (Ljava/util/Map;Lcom/adobe/marketing/mobile/Event;)V
 	public abstract fun createXDMSharedState (Ljava/util/Map;Lcom/adobe/marketing/mobile/Event;)V
 	public abstract fun dispatch (Lcom/adobe/marketing/mobile/Event;)V
+	public abstract fun getHistoricalEvents ([Lcom/adobe/marketing/mobile/EventHistoryRequest;ZLcom/adobe/marketing/mobile/AdobeCallback;)V
 	public abstract fun getHistoricalEvents ([Lcom/adobe/marketing/mobile/EventHistoryRequest;ZLcom/adobe/marketing/mobile/EventHistoryResultHandler;)V
 	public abstract fun getSharedState (Ljava/lang/String;Lcom/adobe/marketing/mobile/Event;ZLcom/adobe/marketing/mobile/SharedStateResolution;)Lcom/adobe/marketing/mobile/SharedStateResult;
 	public abstract fun getXDMSharedState (Ljava/lang/String;Lcom/adobe/marketing/mobile/Event;ZLcom/adobe/marketing/mobile/SharedStateResolution;)Lcom/adobe/marketing/mobile/SharedStateResult;
-	public abstract fun recordHistoricalEvent (Lcom/adobe/marketing/mobile/Event;Lcom/adobe/marketing/mobile/EventHistoryResultHandler;)V
+	public abstract fun recordHistoricalEvent (Lcom/adobe/marketing/mobile/Event;Lcom/adobe/marketing/mobile/AdobeCallback;)V
```

Comment:
Note: This API was added by me in a [previous PR](https://github.com/adobe/aepsdk-core-android/pull/749/files#diff-bf954d4a1545934d2e4eb2b4a3804c1b51f82da4db1dd9dd60bc20dd86a0d927R188-R202) on this feature branch that has not been released yet.